README.md

```diff
@@ -27,7 +29,12 @@ The resort to PyTorch tensors as inputs to Kymatio allows the programmer to back
 
 Each of these algorithms is written in a high-level imperative paradigm, making it portable to any Python library for array operations as long as it enables complex-valued linear algebra and a fast Fourier transform (FFT).
 
-Currently, there are two available backends, PyTorch (CPU and GPU) and scikit-cuda (GPU only).
+As of v0.1, Kymatio fully supports two backends: PyTorch (CPU and GPU) and scikit-cuda (GPU only).
+
+The next version of Kymatio, planned around Spring 2020, will also support NumPy (CPU only) and TensorFlow (CPU and GPU) as backends.
+You may already try out these new backends in the alpha release of Kymatio v0.2, whose source code is available on the following branch: 
+https://github.com/kymatio/kymatio/tree/kymatio-v2
```

Comment:
We should rename this branch to `kymatio-v0.2` before putting this out here, since otherwise it looks like we're referring to v2.0.

Reply:
oof, you're right. Is that a simple thing to do on git?

Reply:
On GitHub, no. I just made a copy of the branch to `v0.2`. We'll have to remove the old one and switch any pending PRs to the new one. Before doing so, is that the name we want to go with? Wasn't there some `0.2.X` naming scheme we were supposed to use?

Reply:
@janden that's a good idea. we can use `0.2.X`. When 0.2 is ready, we merge `0.2.X` onto master. Then we can work towards `v0.2.1` on the `0.2.X` branch ; and in parallel, prepare a `0.3.X` branch for `v0.3.0a0`. OK?